packages/app-providers/src/hooks/useLedger.ts

```diff
@@ -196,11 +196,15 @@ const useLedger = ({ apiPromise, selectedAccount, selectedNetwork }: Params) =>
               ? totalBalance
                   .plus(reservedAmount)
                   .minus(vestedAmountRing)
+                  .minus(ledgerData.stakedRing)
                   .minus(totalDepositsAmount)
                   .minus(unbondedRingAmount)
```

Comment:
This can be removed in next runtime version.

Ref: https://github.com/darwinia-network/darwinia/pull/1010